### ISSUE TITLE
Error while running scan 

### ISSUE THREAD

**Issue by hackerabhinavverma** [cloudsploit/scans#103]
I am getting below error while running for my aws account

```
/Users/abhinav/cplodsploit/scans/node_modules/aws-sdk/lib/request.js:31
            throw err;
            ^

RangeError: Maximum call stack size exceeded
    at /Users/abhinav/cplodsploit/scans/index.js:85:16
    at replenish (/Users/abhinav/cplodsploit/scans/node_modules/async/dist/async.js:836:21)
    at /Users/abhinav/cplodsploit/scans/node_modules/async/dist/async.js:842:29
    at /Users/abhinav/cplodsploit/scans/node_modules/async/dist/async.js:804:16
    at /Users/abhinav/cplodsploit/scans/index.js:112:13
    at /Users/abhinav/cplodsploit/scans/plugins/sns/topicPolicies.js:115:4
    at /Users/abhinav/cplodsploit/scans/node_modules/async/dist/async.js:365:16
    at replenish (/Users/abhinav/cplodsploit/scans/node_modules/async/dist/async.js:831:29)
    at /Users/abhinav/cplodsploit/scans/node_modules/async/dist/async.js:842:29
    at /Users/abhinav/cplodsploit/scans/node_modules/async/dist/async.js:804:16
```

**Comment by Dave3of5** [comment]
Are you running on windows by chance? I solved this by increasing the stack size:

`node --stack_size=2000 index.js`

This doubles the stack size and allows the process to complete. You may need to tweek that value to something different to get it to work on your system.

**Comment by rfletcher** [comment]
I hadn't seen this issue, but opened PR #110 after running into the same problem.

**Comment by matthewdfuller** [comment]
Thanks for reporting @hackerabhinavverma - I've merged #111 which should fix this issue for you. Please let me know if you have any other trouble.
